package.json

```diff
@@ -46,68 +46,65 @@
   },
   "homepage": "https://github.com/GoogleChrome/workbox#readme",
   "devDependencies": {
-    "@google-cloud/storage": "^1.7.0",
-    "@octokit/rest": "^15.8.1",
+    "@babel/core": "^7.1.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
+    "@babel/preset-env": "^7.1.0",
+    "@google-cloud/storage": "^2.0.3",
+    "@octokit/rest": "^15.12.0",
     "@std/esm": "^0.26.0",
```

Comment:
I tried moving to the `esm` project (which replaced `@std/esm`), but there are a lot of breaking changes and I couldn't get our unit test runner happy. So we're still stuck on this older version until we can investigate further.